### DISCUSSION TITLE
homepage link on PyPI not working

### DISCUSSION THREAD

**Discussion by benwandrew** [AutoResearch/autora#366] [Core Contributions] [answered]
the homepage link in our subpackage template is specified with `https` but our site only works with `http`. that is, https://empiricalresearch.ai doesn't work; only http://empiricalresearch.ai does. so homepage under Project Links in the new DARTS (https://pypi.org/project/autora-theorist-darts/) and BSR (https://pypi.org/project/autora-theorist-bsr/) subpackages lead to "This site can't be reached"

<img width="644" alt="Screen Shot 2023-04-27 at 13 54 31" src="https://user-images.githubusercontent.com/28738595/234951081-3b3f53a7-d42a-4bf6-9166-53e7e939be50.png">

**Comment by benwandrew** [comment]
@younesStrittmatter @musslick created a simple PR to fix this: https://github.com/AutoResearch/autora-theorist-template/pull/3

**Comment by benwandrew** [comment]
PR merged into the template (thanks for the quick approval @younesStrittmatter!), but existing subpackages will need to be fixed @hollandjg @musslick

**Comment by hollandjg** [comment]
I think it would be better to make it so that https://www.empiricalresearch.ai works – [I think http://... will be down-weighted in search engines, as it's not secure.](https://searchengineland.com/google-starts-giving-ranking-boost-secure-httpsssl-sites-199446)

**Reply by musslick** [reply]
@hollandjg Could we use the http:// for now? Changing to https turns out to be complicated and requires some time. Would be better to have a working unsafe link than a broken link for now.

**Reply by hollandjg** [reply]
absolutely agree, in fact we might be able to get away with dropping the http:// completely

**Reply by hollandjg** [reply]
nope, it needs the protocol.

**Reply by hollandjg** [reply, marked as answer]
created an issue to track this #376
